src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
 
 import collaborationCommentonAnyPage from 'images/collaboration_screenshot-crop_comment-on-any-page.webp'
 
-You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our integrations with Slack, MS Teams (coming soon), and email let you share links and screenshots and communicate with your teammates directly from the New Relic UI without switching applications. Our collaboration tools allow you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing MTTR.
-
-## How it works [#collaborate]
-
-We've integrated discussion tools directly into New Relic so you can: 
-
-* **Start discussions in context at the source.** When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
-* **Create shared knowledge and history.** Collaborating within the New Relic platform allows your team to build a searchable body of shared knowledge.
+You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our collaboration tools will enable you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing MTTR. When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
 
 ## Comment on any page [#comment-on-any-page]
 
-On the top navigation, there's a **Share** button and menu on most pages. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, select **Share** and then **Start a new discussion**. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
+On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, select **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```

Comment:
```suggestion
On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```

Reply:
@homelessbirds what's the difference between "Share to Slack" and "Start New Discussion"? This paragraph doesn't seem to mention the second option.